mk/waf.mk

```diff
@@ -0,0 +1,106 @@
+# Makefile wrapper to allow old make commands to use waf build system
+
+SYSTYPE			:=	$(shell uname)
+
+ifneq ($(findstring CYGWIN, $(SYSTYPE)),) 
+  MK_DIR := $(shell cygpath -m ../mk)
+else
+  MK_DIR := $(patsubst %/,%,$(dir $(lastword $(MAKEFILE_LIST))))
+endif
+
+include $(MK_DIR)/environ.mk
+
+
+TARGET_BOARD=sitl
+TARGET_BUILD=$(SKETCH)
+WAF_FLAGS=
+FRAME=
+BINARY_FRAME=
+
+ifeq ($(SKETCH),ArduPlane)
+TARGET_BUILD=plane
+BINARY_NAME=arduplane
+endif
+
+ifeq ($(SKETCH),ArduCopter)
+TARGET_BUILD=copter
+BINARY_NAME=arducopter
+FRAME="quad"
+BINARY_FRAME="-quad"
+endif
+
+ifeq ($(SKETCH),APMrover2)
+TARGET_BUILD=rover
+BINARY_NAME=ardurover
+endif
+
+ifeq ($(SKETCH),AntennaTracker)
+TARGET_BUILD=antennatracker
+BINARY_NAME=antennatracker
+endif
+
+FRAMES = quad tri hexa y6 octa octa-quad heli single coax
+BOARDS = px4-v1 px4-v2 px4-v4 sitl flymaple linux vrbrain vrbrain-v40 vrbrain-v45 vrbrainv-50 vrbrain-v51 vrbrain-v52 vrubrain-v51 vrubrain-v52 vrhero-v10 erle pxf navio navio2 raspilot bbbmini minlure erlebrain2 bhat qflight pxfmini
```

Comment:
waf doesn't support all this boards:

- flymaple has been removed from master
- vrbrain isn't supported but should be easy to add (I was waiting for the VRBRAIN PRs to be merged to do it)
- qflight is apparently broken (I think Lucas said it) and there isn't support for it in waf

I think erle is erleboard. Missing bebop and disco.